modules/ding_availability/js/ding_availability_labels.js

```diff
@@ -191,15 +188,6 @@
           if (status[i] === true) {
             class_name = i;
           }
-          else {
```

Comment:
Why are this removed?

Reply:
This was removed because on each ajax request done during one page load this function was triggered and to each non-online item in collection was added all classes (available, unavailable, reservable, not-reservable), fact which also was breaking ding_availability functionality.

Reply:
Ok, but how do items get un-available and not-reserveable now?

Reply:
The classes with `true` value only will be added.

Reply:
I guess that there was a purpose by adding a class called `not-available` to items that is not available, as well as `un-reserveable` to items that can not be reserved.

Now again; what is the purpose of removing this? Does it improve performance, or?

Reply:
I've removed those because:
1. I didn't found any logic there those classes was used, so we got rid of dead code.
2. This condition wasn't working as it was expected. Classes, as I wrote in 1-st reply, was added to each element despite the fact that those classes should not have been added there. Skipping this part, I can treat as an improvement.